src/reference/asciidoc/kafka.adoc

```diff
@@ -339,15 +339,15 @@ Receiver] pattern and Spring Integration provides an
 http://docs.spring.io/spring-integration/reference/html/messaging-endpoints-chapter.html#idempotent-receiver
 [implementation thereof].
```

Comment:
Not related to this PR, but the divided link part brake it.
So the part in the `[]` should be moved to the same row as url.